### DISCUSSION TITLE
LoS in Realistic Multiuser MIMO OFDM Simulations

### DISCUSSION THREAD

**Discussion by yacine074** [NVlabs/sionna#49] [Q&A] [answered]
Hello,

1- I would like to know how to set the topology to work under LoS in [Realistic Multiuser MIMO OFDM Simulations](https://nvlabs.github.io/sionna/examples/Realistic_Multiuser_MIMO_Simulations.html#System-Setup), the goal is to have a single path for each channel between 1 user and 1 BS.
2- Is it possible to increase the number of base stations in this example? 
 
Thank you

**Comment by faycalaa** [comment, marked as answer]
Hi,

1. The 3GPP channel models use more than one paths. If my understanding is correct, you want to keep only the LoS path, and discard the other paths generated by the 3GPP channel model. To do that, you need to:

    1. Force the user to be in LoS. Note that in 3GPP models, a user is in LoS with a probability specified by the specification. When setting the topology using the `set_topology` function of the channel model you are using ([UMi](https://nvlabs.github.io/sionna/api/channel.html#sionna.channel.tr38901.UMi), [UMa](https://nvlabs.github.io/sionna/api/channel.html#sionna.channel.tr38901.UMa), or [RMa](https://nvlabs.github.io/sionna/api/channel.html#sionna.channel.tr38901.RMa)), you can force the user to be in LoS as follows:

        1. Force the user to be located outdoor by setting the `in_state` parameter  to `False` for all batch examples.
        2. Force the user to be in LoS by setting the `los` parameter to `True`.
    
    2. Generate channel impulse responses (CIRs) by calling the channel model. A CIR correspond to path coefficients `a` and delays `tau`.

    3. Keep only the first path coefficient and delay, which define the LoS path.

    4. Use this altered CIR to generate frequency-domain or time-domain channel coefficients using the [cir_to_ofdm_channel](https://nvlabs.github.io/sionna/api/channel.html#sionna.channel.cir_to_ofdm_channel) or [cir_to_time_channel](https://nvlabs.github.io/sionna/api/channel.html#sionna.channel.cir_to_time_channel) function, that can then be applied to channel inputs using [ApplyOFDMChannel](https://nvlabs.github.io/sionna/api/channel.html#sionna.channel.ApplyOFDMChannel) or [ApplyTimeChannel](https://nvlabs.github.io/sionna/api/channel.html#sionna.channel.ApplyTimeChannel) layer.

2. Yes. When setting the topology using the `set_topology` method of the channel models, you can specify multiple base stations locations and orientations.

**Reply by yacine074** [reply]
Thank you for your quick answer and clear explanation.  

I have another question, is there a possibility to make a communication between two base stations?
